specs/v1.yaml

```diff
@@ -522,6 +523,7 @@ paths:
         - The amount must be in whole dollar increments (rounded to the nearest hundred) as currently DAFs only accept whole dollar grants otherwise the request will return status `400 Bad Request`.
         - The amount must be greater than or equal to the minimum grant amount for the DAF otherwise the request will return status `400 Bad Request`.
         - The amount must be less than or equal to the user's DAF account balance otherwise the request will return status `400 Bad Request`.
+        - Any inputs exceeding the maximum allowed length will be automatically truncated.
```

Comment:
i would say: "automatically truncated unless otherwise stated"

Reply:
there might be cases for some fields where we want to fail the request vs. allowing it to just be truncated and succeed

Reply:
```suggestion
        - Any inputs exceeding the maximum allowed length will be automatically truncated unless otherwise stated.
```